container/devel/Dockerfile.openqa.leap

```diff
@@ -0,0 +1,34 @@
+FROM registry.opensuse.org/opensuse/leap:15.6
+
+RUN zypper ar -G https://download.opensuse.org/repositories/devel:/openQA/15.6 devel-openqa \
+        && zypper ar -G https://download.opensuse.org/repositories/devel:/openQA:/Leap:/15.6/15.6/ devel-openqa-leap \
+        && zypper ar -G https://download.opensuse.org/repositories/SUSE:/CA/15.6/SUSE:CA.repo \
+        && zypper -n ref \
+    && true
```

Comment:
What's the purpose of this?

Reply:
The `&&`? Just in my muscle memory to always end multiline commands with this, so it is easy to add a command without having to add a `\` at the previous line :)
I guess I can remove it here